docs/application/tizen-studio/native-tools/leak-sanitizer.md

```diff
@@ -0,0 +1,28 @@
+# Detecting Runtime Memory Leaks with Leak Sanitizer
```

Comment:
Can we use a Sentence case for all pages in this PR?
https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#capitalization
```suggestion
# Detect runtime memory leaks with Leak Sanitizer

Reply:
sure, thank you!